### PR TITLE
Add component for each Role and withdraw text.

### PR DESCRIPTION
**Pull request by iooeduard**

### CODE REVIEW

File: packages/react-app/src/components/WithdrawText/withdraw-text.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+function WithdrawText() {
+  return (
+    <div>
+      <Flex>
+        <Text
+          mt={ 3 }
+          fontSize={ 5 }
+        >
+          Hi, Freak you can withdraw up to: XXXX USDC
```
Comment (dasilvaluis) [inline packages/react-app/src/components/WithdrawText/withdraw-text.jsx]: Maybe you can already define a prop for the value he can withdraw?

File: packages/react-app/src/components/WithdrawText/withdraw-text.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Field, Input, Flex, Text } from 'rimble-ui';
+
+function WithdrawText() {
+  return (
+    <div>
+      <Flex>
```
Comment (dasilvaluis) [inline packages/react-app/src/components/WithdrawText/withdraw-text.jsx]: In this case you can wrap everything in a 

```js
<Flex flexDirection="column">
  <Text>...</Text>
  <Flex>
    <Text>...</Text>
    <Field>...</Field>
  </Flex>
</Flex>
```

File: packages/react-app/src/components/WithdrawText/withdraw-text.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Field, Input, Flex, Text } from 'rimble-ui';
+
+function WithdrawText() {
```
Comment (dasilvaluis) [inline packages/react-app/src/components/WithdrawText/withdraw-text.jsx]: Would it make more sense to call it `WithdrawForm`?

**Review by dasilvaluis — CHANGES_REQUESTED** [review]

File: packages/react-app/src/components/WithdrawText/withdraw-text.jsx (hunk 1 above)
Comment (iooeduard) [reply packages/react-app/src/components/WithdrawText/withdraw-text.jsx]: Yes, good idea.

File: packages/react-app/src/components/WithdrawText/withdraw-text.jsx (hunk 3 above)
Comment (iooeduard) [reply packages/react-app/src/components/WithdrawText/withdraw-text.jsx]: Probably yes, thanks.

**Review by dasilvaluis — APPROVED** [review]

**Comment by iooeduard** [comment]
[UI component for non-freak callers](https://trello.com/c/W1wkEIuM/206-ui-component-for-non-freak-callers)

**Pull request merged**